Gemfile

```diff
@@ -7,8 +7,12 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 # Synchronize with https://pages.github.com/versions
 ruby '>=2.7.1'
 
-gem 'github-pages', group: :jekyll_plugins
+group :jekyll_plugins do
+  gem 'github-pages'
+  # Allows accessing ENV variables in CI environments
+  gem 'jekyll-environment-variables'
```

Comment:
I guess this gem is necessary on GitLab only. If so, could you please make it depend on whether it's executed in GitLab? You can do so by appending `if ENV["SOME_GITLAB_VAR"]` to the end of the line.

Reply:
Apologies, only saw this comment after mine below.
Would it be ok to use the same approach to only load the first plugin if running in GitHub?

Reply:
To be precise, `if ENV["GITHUB_ACTIONS"]` which according to [GitHub Actions environment variables](https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables) is only set when GitHub Actions is running a workflow.

Reply:
In fact, the `jekyll-environment-variables` plugin might be useful in GitHub Actions too if wanting to access any of the env variables mentioned in the previous comment.

Reply:
> Apologies, only saw this comment after mine below.
> Would it be ok to use the same approach to only load the first plugin if running in GitHub?

Hum.. this won't work. Making both plugins conditional causes jekyll to not be installed at all.